adafruit_veml7700.py

```diff
@@ -37,6 +37,12 @@
 from adafruit_register.i2c_bits import RWBits
 from adafruit_register.i2c_bit import RWBit, ROBit
 
+try:
+    import typing  # pylint: disable=unused-import
+    from busio import I2C
```

Comment:
Should the import of `busio` be outside the `try`…`except` block, or should these imports be reversed? If you’re running this on a CPY device, the import of `typing` will fail and `busio.I2C` won’t get imported. I don’t have a device handy to test with at the moment, but that intuitively seems like it would be a problem to me.

Reply:
So, you can check with @FoamyGuy  or @tekktrik   my understanding is the import from typing is not available in circuitpython, so it is a ‘guard statement’ that forced the entire try block to fail. This keeps the busio statement from interfering with CP code, but allows ide s to correctly interpret the type annotations

Reply:
Since CircuitPython discards annotations at runtime, there's no issue with things used purely for typing to be unbound at that time.  Since importing nonetheless uses RAM beforehand, we using the `typing` module to guard a `try`/`except` block to prevent the CircuitPython implementation from doing the imports, while allowing the CPython (or other) implementation to import them.